src/observer_cli.erl

```diff
@@ -43,7 +44,13 @@ start(Node, Cookie) when is_atom(Node) andalso is_atom(Cookie) ->
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 rpc_start(Node) ->
     case net_kernel:connect_node(Node) of
-        true -> rpc:call(Node, ?MODULE, start, [#view_opts{}]);
+        true ->
+            {ok, TerminalRow} =
+                case io:rows() of
+                    {error, _} -> rpc:call(Node, io, rows, []);
```

Comment:
Always failed due to `io:rows() =:= rpc:call(Node, io, rows, [])`
just return a default integer.